src/python/grpcio_tests/tests_aio/unit/server_test.py

```diff
@@ -43,18 +46,32 @@ def __init__(self):
     async def _block_forever(self, unused_request, unused_context):
         await asyncio.get_event_loop().create_future()
 
-    async def _BLOCK_BRIEFLY(self, unused_request, unused_context):
+    async def _block_briefly(self, unused_request, unused_context):
         await asyncio.sleep(test_constants.SHORT_TIMEOUT / 2)
         return _RESPONSE
 
+    async def _unary_stream_async_gen(self, unused_request, unused_context):
+        for _ in range(_NUM_STREAM_RESPONSES):
+            yield _RESPONSE
+
+    async def _unary_stream_reader_writer(self, unused_request, context):
```

Comment:
Can we test that there's an exception when you try to mix the async generator style with the `read`/`write` style?

Reply:
Added a mixed test case. For now, I actually think it is fine for users to mix them. As discussed with Jan, it is the users' call to decide which API to use, and it is their responsibility to handle synchronization.

Reply:
Nice! Being able to mix the two styles is great as long we make sure to thoroughly test it.